tor/__init__.py

```diff
@@ -1,49 +1 @@
-import bugsnag
-from addict import Dict
-
 __version__ = '2.6.14'
```

Comment:
Just gave it a read through; I'm fine with that implementing that. Seems like a sane way to do it :)